shared/elements/gaia_pin_card/script.js

```diff
@@ -88,7 +88,9 @@ window.GaiaPinCard = (function(win) {
   var template = document.createElement('template');
   template.innerHTML =
     `<article class="pin-card no-content">
-      <i></i>
+      <div class="icon-container">
+        <icon></icon>
```

Comment:
I remember reading that using <i> for icons was not the best idea (semantically),  but not sure where the icon tag came from. Let's leave it as it was. Thanks!